cmd/cored/main.go

```diff
@@ -233,7 +234,20 @@ func launchConfiguredCore(ctx context.Context, db *sql.DB, conf *config.Config,
 		if err != nil {
 			chainlog.Fatal(ctx, chainlog.KeyError, err)
 		}
-		s := blocksigner.New(blockPub, mockHSM, db, c) // TODO(jackson): support real HSM
+
+		var hsm blocksigner.Signer = mockHSM
+		if conf.BlockHSMURL != "" {
+			hsm = &remoteHSM{Client: &rpc.Client{
+				BaseURL:      conf.BlockHSMURL,
+				AccessToken:  conf.BlockHSMAccessToken,
```

Comment:
signerd only uses the password portion of the auth. seems like there are two options:

1) take a "access token" string of "username:password" when configuring (in this case ":password") and don't worry about modifying it
2) take a password when configuring, and convert to an access token string here